packages/extension-url-mapping/src/setup-app.ts

```diff
@@ -131,9 +132,15 @@ class UrlMapping extends ApplicationExtension<Config> {
         // Jinsu's work on serialization will make sure that the same page will be rendered on both server and client sides.
 
         const result = [myRoute, ...routes]
-        console.log('--- beforeRouteMatch: resulting routes', result)
+        // console.log('--- beforeRouteMatch: resulting routes', result)
         return result
     }
+
+    getComponentMap(): ComponentMap {
+        return {
+            Foo: SamplePage
+        }
+    }
```

Comment:
In my PR, I was planning to remove the temporary extension-url-mapping because it was just for our testing/exploring. But now I think that I'll let you remove the extension in your PR instead.